### DISCUSSION TITLE
Is the "CommunityToolkit.Mvvm" supported?

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#10204] [Q&A] [answered]
[_From Discord_:](https://discord.com/channels/372137812037730304/550416151172087808/1032318738223071373)

Is the "CommunityToolkit.Mvvm" not fully supported by UNO? [I did not see it here](https://github.com/unoplatform/uno/blob/master/doc/articles/supported-libraries.md).

**Comment by jeromelaban** [comment, marked as answer]
The CommunityToolkit.Mvvm package[ is a `netstandard2.0` package](https://nuget.info/packages/CommunityToolkit.Mvvm/8.0.0), so there's no need for an Uno-specific version. You can use is as-is without any restriction, on all targets Uno supports.

**Reply by jpgarza93** [reply]
I saw this in an UNO Platform youtube video
![image](https://user-images.githubusercontent.com/79015116/196816552-6b0cc578-0b62-4dee-98db-9b7c175448db.png)

**Reply by jpgarza93** [reply]
thanks

**Reply by ivanjx** [reply]
wait so i can use all controls from vendors like syncfusion in uno?
